### DISCUSSION TITLE
Tabs display the content of Tabs.Panel for SEO

### DISCUSSION THREAD

**Discussion by florianfranconi** [tailwindlabs/headlessui#740] [Ideas]
Hi, I use Tabs and everything works fine. But I noticed that only the selected Panel is displayed in the DOM, which is a problem for SEO. It would be nice to have a props to display all the Panels in the DOM and to `display: none;` of those that are not selected. This would allow Google's indexing robots to properly crawl all the content of the Tabs.

Uber uses a props `renderAll` on these Tabs to counter this problem: https://baseweb.design/components/tabs-motion#rendering-and-seo

Thanks

**Comment by RobinMalfait** [comment]
Hey!

That should already be possible by passing an `unmount={false}` (React) or `:unmount="false"` (Vue) prop to the Tab Panel component.
I will take note to add it to the docs!

**Reply by florianfranconi** [reply]
Indeed it works, thank you!

**Reply by msmosavar** [reply]
Thanks! Saved me hours...
But there is nothing mentioned in the docs yet!

**Reply by andcl** [reply]
Still missing in the docs :( Lost a couple of hours until I arrived here...

**Reply by joemaffei** [reply]
Confirmed working in Vue and undocumented as of today.

**Reply by joemaffei** [reply]
@RobinMalfait Is it possible to make contributions to the documentation?

**Reply by Minho-Lee** [reply]
Still no documentation on https://headlessui.com/react/tabs ... 😢
